apps/arena/native/physics/src/collision_detection/sat.rs

```diff
@@ -245,7 +257,7 @@ fn project_circle(circle: &Entity, axis: Position) -> (f32, f32) {
         y: axis.y * circle.radius,
     };
 
-    let position_plus_radius = Position::add(circle.position, direction_radius);
+    let position_plus_radius = Position::add(&circle.position, &direction_radius);
```

Comment:
Since the add function is receiving a reference, lets make all other Positions functions receive a reference as parameter, to have our code consistent

Reply:
sounds good! changed